distributed/tests/test_client.py

```diff
@@ -7262,6 +7263,9 @@ def fetch_comp_code(dask_scheduler):
 
 def test_computation_object_code_not_available(client):
     np = pytest.importorskip("numpy")
+    if parse_version(np.__version__) >= parse_version("1.25"):
+        raise pytest.skip("numpy >=1.25 can capture ufunc code")
```

Comment:
I've never seen `raise pytest.skip(...)` used before, only `pytest.skip(...)`. Clearly the `raise` works -- for my own understanding, is there a benefit to adding the `raise` or just syntactic preference?

Reply:
Nope, just my typo. Incidentally the code works because the `raise` statement is never reached. I'll amend it.

Reply:
https://github.com/dask/distributed/pull/7939